lib/bintray_client.rb

```diff
@@ -0,0 +1,53 @@
+require 'net/http'
+require 'json'
+
+class BintrayClient
+
+  def initialize(user, key, base_url = 'https://bintray.com/api/v1/packages')
+    @base_url = base_url
+    @user = user
+    @key = key
+  end
+
+  def get_bintray_versions(project)
```

Comment:
Would be nice to have rubydoc comments for all the methods of all classes in `lib/`